src/custom-addons/orr_document_extend/models/ir_attachment.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields,api,_
+from odoo.exceptions import UserError
+
+class IrAttachment(models.Model) :
+
+    _inherit = "ir.attachment"
+    @api.model
```

Comment:
Better usage of newlines
```suggestion
    _inherit = "ir.attachment"

    @api.model
```